scripts/gnomad-json.py

```diff
@@ -0,0 +1,177 @@
+import pandas as pd
+import json
+from statsmodels.stats import proportion
```

Comment:
Could you please add these dependencies to `scripts/environment.yml`? This will make it easier to add to the Snakemake

Reply:
Sure!

Reply:
Should I make the input and output as command line inputs? They'll be positional arguments with first as the tsv and the json.

Reply:
Yep, that would work. I just did a simple argparse CI for the other python scripts.